packages/utils/src/accessibility/helpers.ts

```diff
@@ -1,8 +1,32 @@
+import { css as litCSS } from 'lit';
+
 /**
  * When multiple ids provided, separate by SPACE
  */
 const SEPARATOR = ' ';
 
+/**
+ * get a style hiding elements visually with `.visually-hidden` selector.
+ * These elements would be available to screen readers only.
+ * @param css lit's css literal template tag
+ * @returns CSSResult
+ */
+const visuallyHiddenStyle = (css: typeof litCSS) => {
```

Comment:
I think it should not have the `css` param.

Reply:
updated with constant + `unsafeCSS()`.